app/views/layouts/application.html.erb

```diff
@@ -13,6 +13,15 @@
       <div class="navbar-header">
         <%= link_to 'Policial.io', root_path, class: 'navbar-brand' %>
       </div>
+      <div class="navbar-right">
+	<% if current_user %>
+	  <%= link_to "#" do %>
```

Comment:
yeah.. i thought so too.. not sure how to CSS that tho.. we can leave that for a next iteration .